src/controllers/menu.controller.ts

```diff
@@ -48,10 +66,25 @@ export const createMenu = async (req: Request, res: Response) => {
 
 export const updateMenu = async (req: Request, res: Response) => {
     try {
-        const menuToUpdate = await Menu.findOneBy({ id: +req.params.id });
+        const menuToUpdate = await Menu.findOne({
+            where: { id: +req.params.id },
+            relations: { recipes: true, },
+        });
+        const { name, recipes } = req.body;
         if (!menuToUpdate) throw new Error(`Menu with id of ${+req.params.id} can not be found!`)
-        await Menu.update(menuToUpdate.id, req.body);
-        const menuUpdated = await Menu.findBy({ id: +req.params.id })
+        menuToUpdate.name = name;
+        const categories = req.body.categories;
+        const menuRecipes = Array();
+        for (const category of categories) {
```

Comment:
This segment looks the same to the create method. Do we need to extra into a helper method?

Reply:
Will solve this in a separate issue/